code/modules/mining/lavaland/necropolis_chests.dm

```diff
@@ -563,7 +568,27 @@
 	..()
 
 
+/obj/item/jacobs_ladder
+	name = "jacob's ladder"
+	desc = "A celestial ladder that violates the laws of physics, but respects the coordinate plane."
```

Comment:
coordinate plane is not an IC friendly term

Reply:
Its a player visible/in universe thing and is what is used for the GPS/suit sensors/telesci etc, I'll switch to a different term if anyone has one though

Reply:
"quadrant"